docker-compose.yml

```
@@ -13,5 +13,15 @@ services:
       POSTGRES_PASSWORD: postgres
       POSTGRES_DB: cpsc304_dev
 
+  backend:
+    container_name: cpsc304-backend
+    build:
+      context: ./backend
+    depends_on:
+      - postgres
+    expose:
+      - "8080"
```

With this change I'm able to connect to the server via my WSL2's IP. I don't know anything about Docker though so maybe there's a better way? 🤷‍♂️

```suggestion
      - "8080"
    ports:
      - "8080:8080"
```

---

my bad, you're right 
difference between expose and ports below:
https://stackoverflow.com/questions/40801772/what-is-the-difference-between-docker-compose-ports-vs-expose

I forget that we are using the 'backend" to serve our frontend as well, we should change the naming convention to "web" instead for all related folders and files

---

@nuxeric Good idea! Created #4 to make that change quickly after these are merged

---

@nuxeric  Would this make `expose` redundant here then? And would it still be needed in the Dockerfile?